### DISCUSSION TITLE
Persistent clients and Runtime clients

### DISCUSSION THREAD

**Discussion by muchqs** [AdguardTeam/AdGuardHome#5466] [Q&A]
In AGH -> Settings -> Client Settings
Under "Runtime clients", it says "Devices not on the list of Persistent clients that may still use AdGuard Home". But then it lists all the clients set under the "Persistent clients" list. Why?

**Comment by nareddyt** [comment]
Filed https://github.com/AdguardTeam/AdGuardHome/issues/5985
